lib/base/RequestClient.js

```diff
@@ -94,6 +95,11 @@ RequestClient.prototype.request = function(opts) {
   this.lastRequest = new Request(optionsRequest);
 
   axios(options).then((response) => {
+    if (opts.debug === true) {
+      const requestLog = `Request: ${response.config.method.toUpperCase()} | ${response.config.url} | ${JSON.stringify(response.config.data)} | ${JSON.stringify(response.config.headers)}`
```

Comment:
maybe try writing a helper function and match the style to the CLI? https://github.com/twilio/twilio-cli-core/blob/8748ddb3d1158ffcff40e3fe022977574f301cb4/src/services/cli-http-client.js#L118